src/Uno.UI.Composition/Composition/Visual.skia.cs

```diff
@@ -143,4 +154,51 @@ private protected DrawingSession BeginDrawing(SKSurface surface, in DrawingFilte
 
 		return session;
 	}
+
+	internal void RedirectTouchForPointerId(uint pointerId)
```

Comment:
Sounds not really related to a `Visual`, I would try to not add such methods here to avoid to end with a too massive class.